src/main/java/alice/command/CommandFind.java

```diff
@@ -0,0 +1,49 @@
+package alice.command;
+
+import alice.Alice;
+import alice.task.Task;
+
+import java.util.List;
+import java.util.Locale;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class CommandFind extends Command {
+
+	private static final String SUCCESS_MESSAGE = "Here are the matching tasks in your list:\n%s";
+
+	private static final String USAGE = "find Usage: find [string]";
+
+	public CommandFind(String[] tokens) {
+		super(tokens);
+	}
+
+	@Override
+	public Alice execute(Alice agent) {
+		try{
```

Comment:
perhaps there could be a spacing here? like try { instead of try{

but I'm pretty sure this is just a careless mistake so don't worry too much about it!